app/assets/stylesheets/application.scss

```diff
@@ -180,7 +184,7 @@ iframe{
 }
 
 .footer .social {
-  color: white;
+  color: $white;
```

Comment:
I know the styleguide says to do this, but something I find disturbing about using the color's name as your color variable is that if you are changing your text color ever, you have to go to EVERY instance and update that